test/pagespecs/AccessibilityStatementPageSpec.scala

```diff
@@ -21,18 +21,4 @@ import testsupport.stubs._
 
 class AccessibilityStatementPageSpec extends ItSpec {
```

Comment:
Suggest here to either:
a) Remove this suite entirely and remove the `accessibilityStatementPage` as well
or b) update the existing test to just check that the accessibility link is present and points at the right place